packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -19,12 +19,14 @@ moduleFor(
         // internally. Specifically, it returns a promise when transitioning _into_
         // the first engine route, but returns the synchronously available handler
         // _after_ the engine has been resolved.
-        _getHandlerFunction() {
-          let syncHandler = this._super(...arguments);
+        setupRouter() {
+          this._super(...arguments);
+          let syncHandler = this._routerMicrolib.__proto__.getHandler;
```

Comment:
I don’t fully grok why we have to pull this from `__proto__`, can you help me understand?